collect_app/src/main/res/values/attrs.xml

```diff
@@ -8,5 +8,7 @@
         <attr name="warningColor" format="color" />
         <attr name="rankItemColor" format="color" />
         <attr name="dividerCompat" format="reference" />
+
+        <attr name="colorOnSurfaceMediumEmphasis" format="color" />
```

Comment:
This was an interesting one: As Android doesn't have a way to change emphasis/opacity on `TextView`s without changing the full color it made sense to define a medium emphasis version of the standard `colorOnSurface` (what would be the base color for text on the white surface) attribute. The value (set in `theme.xml`) is set to 60% which corresponds with the working [color palette](https://github.com/opendatakit/collect/files/3382372/collect-styleguide.pdf).